src/reader/TranslatableWord.js

```diff
@@ -9,6 +9,7 @@ export default function TranslatableWord({
   pronouncing,
   translatedWords,
   setTranslatedWords,
+  isBookmarkWord,
```

Comment:
this could stay, if we renamed it to `disableTranslation` - so it still is generic.